src/framework/preferences/js/Panels.js

```diff
@@ -841,7 +841,7 @@ var fluid_2_0 = fluid_2_0 || {};
             theme: ["default", "bw", "wb", "by", "yb", "lgdg"]
         },
         markup: {
-            label: "<span class=\"fl-preview-A\">A</span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
+            label: "<span class=\"fl-preview-A\"></span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
```

Comment:
@jobara Tested nvda on all the browsers and it seems like the only one that is reading the "a" is firefox. gonna do some research see if there's a quick workaround before trying @klown 's suggestion